### ISSUE TITLE
LegitMode Setting

### ISSUE THREAD

**Issue by TexRexTex** [cabaletta/baritone#1048]
## Describe your suggestion
Enables LegitMine and prevents the player from walking backwards also enables the [gui](https://github.com/cabaletta/baritone/issues/1047) setting. makes the player not look in sharp directions and instead makes it look like the player's mouse is moving and will be slightly randomized so the looking speed will be different every time.

## Settings
Enabled/Disabled

## Context
This would make Baritone look more legitimate and therefore harder for admins/staff to catch a user using baritone.

## Final checklist
- [✓] I have not used any OwO's or UwU's in this issue.

**Comment by mrkvicka22** [comment]
Yep looking into direction of your walking would be fine. Also turning your head slower in legit mine mode.

**Comment by Xephorix** [comment]
another thing that could add to the realism would be to randomly change the speed at which you change the direction you're facing. so it seems like a person is controlling it. because turning your head slowly but smoothly would still be a bit suspicious.

**Comment by TexRexTex** [comment]
@Xephorix yes I agree I think that would be a good addition

**Comment by Pigsti123** [comment]
Yeah that sounds like a cool idea...Definitely need that on the mod. Thanks for sharing!

**Comment by mrphax0r** [comment]
The pack of suggestions here is great, in my opinion. 
Smooth head rotations, jitter, and randomization are things we need.

**Comment by JacksonChen666** [comment]
disabling freecam in the baritone settings makes baritone look in the direction it is walking to

**Comment by TheK0tYaRa** [comment]
Not that it helps to be less seen by the admins of any server.

**Comment by mrphax0r** [comment]
@TheK0tYaRa It does. The admins of the servers have usually dealt with a few baritone users. If you as a mod see somebody doing things as suspiciously as baritone currently does, you'll instantly ban them. Just look at your character in freecam while pathing, it's pretty clear.

**Comment by TheK0tYaRa** [comment]
@mrphax0r Let me rephrase: if the player turns 45 degrees with or without his head turning it's still suspicious. I'm not saying "issue is useless", i'm saying "set freecam true is useless" for the case of avoidance of being banned.

**Comment by l1ving** [comment]
Closing this in favor of #2034 as it has more detail.

**Comment by Moongrassy** [comment]
yea my friend keep says he is going to ban me so i'm trying to make my head turn slower so he does not ban me he's like Moongrassy why is your head turning so fast and then i go silent

**Comment by scorbett123** [comment]
> yea my friend keep says he is going to ban me so i'm trying to make my head turn slower so he does not ban me he's like Moongrassy why is your head turning so fast and then i go silent

Don't use baritone on servers which it isn't allows
